Library/AppEnvironmentTests+OAuthInKeychain.swift

```diff
@@ -0,0 +1,422 @@
+@testable import KsApi
+@testable import Library
+import XCTest
+
+final class AppEnvironmentTests_OAuthInKeychain: XCTestCase {
```

Comment:
These are all tests run in the hosted environment, so they can actually test the keychain. I included a bunch of duplicates of tests in `AppEnvironmentTests` that just swap the feature flag to 'on' to correctly test that behavior.